makefile

```diff
@@ -271,6 +271,9 @@ tests: opt-tests unit-tests lower-tests quine-tests repl-test module-tests doc-f
 unit-tests:
 	$(STACK) test --work-dir .stack-work-test $(STACK_FLAGS)
 
+watch-tests:
```

Comment:
`watch-unit-tests`? Since we use `tests` to mean all tests, not just those.

Reply:
Done.